redhat-cop/automate_tower_genie_collections
# Add the Workflow Job Template Nodes into the generated Workflow Job Template yaml file

### What does this PR do?
When exporting the workflow job templates, this PR is putting it's nodes in the `simplified_workflow_nodes` variable, so it's compatible with the `dispatch` role.

### How should this be tested?
```
$ cat filetree_create.yml
---
- hosts: localhost
  connection: local
  gather_facts: false
  vars:
    controller_hostname: "{{ vault_controller_hostname }}"
    controller_username: "{{ vault_controller_username }}"
    controller_password: "{{ vault_controller_password }}"
    controller_validate_certs: "{{ vault_controller_validate_certs }}"
    controller_api_plugin: "ansible.controller.controller_api"
  pre_tasks:
    - name: "Check if the required input variables are present"
      assert:
        that:
          - input_tag is defined
          - (input_tag | type_debug) == 'list'
        fail_msg: 'A variable called ''input_tag'' of type ''list'' is needed: -e ''{input_tag: [organizations, projects]}'''
        quiet: true

    - name: "Check if the required input values are correct"
      assert:
        that:
          - tag_item in valid_tags
        fail_msg: "The valid tags are the following ones: {{ valid_tags | join(', ') }}"
        quiet: true
      loop: "{{ input_tag }}"
      loop_control:
        loop_var: tag_item
  roles:
    - redhat_cop.controller_configuration.filetree_create
...
$ ansible-playbook filetree_create.yml -e '{input_tag: [workflow_job_templates], output_path: ~/iam/filetree_create.2.1.7 }'
```

### Is there a relevant Issue open for this?
No issue has been opened for this.

### Other Relevant info, PRs, etc.
N/A
